cider/tests/CiderNewHashTableTest.cpp

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "CiderAggTestHelper.h"
+#include "TestHelpers.h"
+#include "cider/CiderTypes.h"
+#include "common/hashtable/FixedHashMap.h"
+
+#ifndef BASE_PATH
+#define BASE_PATH "./tmp"
+#endif
+
+using namespace TestHelpers;
+
+static const std::shared_ptr<CiderAllocator> allocator =
+    std::make_shared<CiderDefaultAllocator>();
+
+class CiderNewHashTableTest : public ::testing::Test {
+ protected:
+  static MockTable* MockTableForTest;
+
+  static void SetUpTestSuite() {
```

Comment:
do we need this?

Reply:
It will be used later, let me remove it now and add back in the future.